client/src/legacy/TabSet.js

```diff
@@ -17,6 +17,39 @@ $.entwine('ss', function($){
 
       this._super();
     },
+    
+    onadd: function () {
+        this.on(
+            "tabsactivate",
+            function (event, { newPanel }) {
+                this.triggerLazyLoad(newPanel);
```

Comment:
```suggestion
                this.lazyLoadGridFields(newPanel);
                this.triggerLazyLoad(newPanel);
```
And remove the `tabsactivate` event handler in the `.ss-tabset`-exclusive entwine scope